website/docs/cli/commands/init.html.md

```diff
@@ -157,6 +157,7 @@ You can modify `terraform init`'s plugin behavior with the following options:
   You can use `-plugin-dir` as a one-time override for exceptional situations,
   such as if you are testing a local build of a provider plugin you are
   currently developing.
+- `-lockfile=MODE` Set dependency lockfile mode. Currently only "readonly" is valid.
```

Comment:
It would be great to get some more detail on why you'd want to use this flag, and what the consequences are.

Reply:
Fixed.